### PR TITLE
PM-14184: Update the switch thoughout the app

### PR DESCRIPTION
**Pull request by david-livefront**
## 🎟️ Tracking

[PM-14184](https://bitwarden.atlassian.net/browse/PM-14184)

## 📔 Objective

This PR updates the switch throughout the app to match the v3 designs.

## 📸 Screenshots

|Before | After |
| --- | --- |
|  |  |
|  |  |
|  |  |

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation or informed the documentation team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed
  issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

[PM-14184]: https://bitwarden.atlassian.net/browse/PM-14184?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/src/main/java/com/x8bit/bitwarden/ui/platform/components/toggle/BitwardenWideSwitch.kt
Comment (SaintPatrck) [inline app/src/main/java/com/x8bit/bitwarden/ui/platform/components/toggle/BitwardenWideSwitch.kt:1]: Can we update this file name to `BitwardenSwitch`?

File: app/src/main/java/com/x8bit/bitwarden/ui/platform/components/toggle/BitwardenWideSwitch.kt [hunk 1]
```diff
@@ -112,26 +125,43 @@ fun BitwardenWideSwitch(
     }
 }
 
-@Preview
+@Preview(showBackground = true)
 @Composable
-private fun BitwardenWideSwitch_preview_isChecked() {
-    BitwardenTheme {
-        BitwardenWideSwitch(
+private fun BitwardenWideSwitch_preview() {
```
Comment (SaintPatrck) [inline app/src/main/java/com/x8bit/bitwarden/ui/platform/components/toggle/BitwardenWideSwitch.kt:130]:
```suggestion
private fun BitwardenSwitch_preview() {
```

File: app/src/main/java/com/x8bit/bitwarden/ui/platform/components/toggle/BitwardenWideSwitch.kt
Comment (david-livefront) [reply app/src/main/java/com/x8bit/bitwarden/ui/platform/components/toggle/BitwardenWideSwitch.kt:1]: Yes!

**Review by SaintPatrck — APPROVED** [review]

**Comment by david-livefront** [comment]
Thanks @SaintPatrck

**Pull request merged**